WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Status/JetpackBackupStatusViewController.swift

```diff
@@ -27,6 +29,28 @@ class JetpackBackupStatusViewController: BaseRestoreStatusViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        coordinator = JetpackBackupStatusCoordinator(site: self.site, view: self)
```

Comment:
Perhaps we can move this line to the `init`?

Reply:
😄 Fixed! b787610